infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -316,9 +263,9 @@ kube-prometheus-stack:
         allow_sign_up: "true"
         client_id: grafana
         client_secret: GRAFANASECRET-CHANGEME
-        api_url: "http://LOCAL-IP:31059/realms/quantumserverless/protocol/openid-connect/userinfo"
-        auth_url: "http://LOCAL-IP:31059/realms/quantumserverless/protocol/openid-connect/auth"
-        token_url: "http://LOCAL-IP:31059/realms/quantumserverless/protocol/openid-connect/token"
+        api_url: "http://RELEASE_NAME-keycloak:31059/realms/quantumserverless/protocol/openid-connect/userinfo"
```

Comment:
I wasn't sure about this @akihikokuroda @psschwei so you can correct me if I'm wrong but I think we can use the service internally? Let me know what you think.

Reply:
I think that I tried but it didn't work. The uses`http://RELEASE_NAME-keycloak:31059` as the base address for the redirect.  It must be an address that the browser can access to.